ztp/gitops-subscriptions/argocd/README.md

```diff
@@ -161,6 +161,28 @@ After all policies become complaint, `ztp-done` label will be added to the clust
 To remove a site and the associated installation and configuration policy CRs by removing the SiteConfig & PolicyGenTemplate file name from the kustomization.yaml file. The generated CRs will be removed as well.
 **NOTE: After removing the SiteConfig file, if its corresponding clusters stuck in the detach process check [ACM page](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.4/html/clusters/managing-your-clusters#remove-managed-cluster) how to clean detach managed cluster **
 
+### Remove obsolete content
+If a change to PolicyGenTemplate configuration results in obsolete policies, for example by renaming policies, the steps in this section should be taken to remove those policies in an automated way.
+
+1. Remove the affected PolicyGenTemplate(s) from GIT, commit and push to the remote repository.
+1. Wait for the changes to synchronize through the application and the affected policies to be removed from the hub cluster.
+1. Add the updated PolicyGenTemplate(s) back to GIT, commit and push to the remote repository.
+
+The following steps can be used to restore files deleted in a previous commit. Once reverted you may make the desired changes to the PolicyGenTemplate files, committing those changes, and then push them to the remote repository.
+```
+    $ git log
+    # Note the hash of the commit which deleted the
+    # PolicyGenTemplates files you want to restore
+    $ git revert <hash>
```

Comment:
This could be dangerous if the commit has more in it than just what they want to restore...

Reply:
Maybe better to not document how to do this? I was hoping to highlight that GIT makes it easy to get stuff back, but your point is a good one. Maybe document with "git revert -n <hash>" to skip the commit and recommend they review the content?